tools/build-easefunction-graphs/src/main.rs

```diff
@@ -0,0 +1,125 @@
+//! Generates graphs for the `EaseFunction` docs.
+use bevy_math::curve::{CurveExt, EaseFunction, EasingCurve};
+use svg::{
+    node::element::{self, path::Data},
+    Document,
+};
+
+fn main() {
+    let directory = "../../crates/bevy_math/images/easefunction";
+    if !std::fs::exists(directory).unwrap() {
+        eprintln!("{directory} does not exist");
+        eprintln!("Make sure to run this from the tools/build-easefunction-graphs folder");
+        std::process::exit(1);
+    }
```

Comment:
could you make this relative to `CARGO_MANIFEST_DIR` so that it doesn't matter where you run it?